src/go/k8s/PROJECT

```diff
@@ -25,4 +25,17 @@ resources:
   kind: Console
   path: github.com/redpanda-data/redpanda/src/go/k8s/apis/redpanda/v1alpha1
   version: v1alpha1
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: com
+  group: redpanda
+  kind: Topic
+  path: github.com/redpanda-data/redpanda/src/go/k8s/apis/redpanda/v1beta1
+  version: v1beta1
```

Comment:
still think this should be v1alpha, so maybe v1alpha2? if you need it not to be v1alpha1

Reply:
Yeah, we don't want beta because that comes with different guarantees to anybody in the k8s world that's familiar with the versioning scheme.

Reply:
Changed to v1alpha2